core/src/stores/sqlite.rs

```diff
@@ -382,6 +384,53 @@ impl Store for SQLiteStore {
         .await?
     }
 
+    async fn load_runs(&self, project: &Project, run_ids: Vec<String>) -> Result<Vec<Run>> {
+        let project_id = project.project_id();
+
+        let pool = self.pool.clone();
+
+        tokio::task::spawn_blocking(move || -> Result<Vec<Run>> {
+            let c = pool.get()?;
+            let run_ids_values = std::rc::Rc::new(
+                run_ids
+                    .iter()
+                    .map(|s| rusqlite::types::Value::Text(s.to_string()))
+                    .collect::<Vec<rusqlite::types::Value>>(),
+            );
+            let mut stmt = c.prepare(
```

Comment:
Same question here

Reply:
Note: checked that we are good indexes wise (and that's the case since we have a unique index on run_id)